src/annotator/config/index.js

```diff
@@ -1,12 +1,57 @@
 import settingsFrom from './settings';
 import { toBoolean } from '../../shared/type-coercions';
 
+/**
+ * Total list of configuration keys.
```

Comment:
```suggestion
 * Complete list of configuration keys.
```

"Total" to me implies a sum, whereas here you mean the "complete set" or "full set".